content/specs/versions-macos.md

```diff
@@ -3,7 +3,7 @@ description: A list of tools available out-of-the-box on Codemagic macOS build m
 title: macOS build machine specification
 aliases:
   - "../releases-and-versions/versions-linux"
-weight: 2
+weight: 1
```

Comment:
I noticed currently we have Linux specs above macOS and I think it makes sense to put macOS first as it contains the default image
Cc @helinanever

Reply:
Agree